.stylelintrc.js

```diff
@@ -79,5 +86,25 @@ module.exports = {
         ],
       },
     ],
+    // Ignore rule until all existing selectors can be updated.
+    'scss/selector-no-union-class-name': null,
+    // Ignore rule until all existing classes can be updated to use BEM.
+    'selector-class-pattern': null,
+    // Allow more specificity until styles can be updated to match the more strict rules.
+    'selector-max-specificity': '0,6,3',
+    // Ignore rule until we confirmed we prefer shorthand properties for positioning.
+    'declaration-block-no-redundant-longhand-properties': null,
   },
+  overrides: [
+    // Always ignore some rules in vendor overrides as we have no control over the way these are written.
+    {
+      files: ['client/scss/overrides/**/*.scss'],
+      rules: {
+        'max-nesting-depth': 5,
+        'selector-attribute-name-disallowed-list': null,
+        'selector-class-pattern': null,
+        'selector-max-combinators': 5,
```

Comment:
I see only two of those rules currently raising issues, and only in two files from our overrides, so I think we’d be better off with in-file overrides here.

Reply:
Good call thank you.